packages/components/src/components/rating-stars/rating-stars.tsx

```diff
@@ -141,37 +191,52 @@ export class RatingStars {
         this.precision
       ),
       0,
-      this.numOfStars
+      this.max
     );
     return star;
   }
 
+  getValueFromXCoordinate(coordinate: number) {
```

Comment:
This is just a suggestion because it doesn't make the component better, only the code :)

Maybe we can also use this function to handle the mouse input? The code is more or less the same as in `getValueFromMousePosition`.

Reply:
Mouse and Touch are now using `getValueFromXPosition(evTou?: TouchEvent, evKey?: MouseEvent)` This function could easily have one parameter only (ev: any), but strange enough some unit test is in need of the explicit type (Mouseevent). It is an alternative to rewrite the function with one parameter only and to drop this very unit test

Reply:
I think the function is fine with the two params!

`evTou` looks a bit funny 😄, again explicitness is preferred so I would have used `evTouch` or even `touchEvent`. But we can leave this like that.